packages/commonwealth/server/routes/status.ts

```diff
@@ -19,6 +23,13 @@ import { success } from '../types';
 import type { RoleInstanceWithPermission } from '../util/roles';
 import { findAllRoles } from '../util/roles';
 
+dotenv.config();
+
+const KNOCK_SIGNING_KEY = process.env.KNOCK_SIGNING_KEY;
```

Comment:
We should not be loading the env var directly in `status.ts`. This variable should be defined in `config.ts` so that we don't import `dotenv` here.